crates/transaction-pool/src/validate/eth.rs

```diff
@@ -237,7 +237,9 @@ where
 
         // Drop non-local transactions with a fee lower than the configured fee for acceptance into
         // the pool.
-        if (!origin.is_local() || self.local_transactions_config.no_local_exemptions()) &&
+        if (!origin.is_local() ||
+            self.local_transactions_config.no_local_exemptions() ||
+            self.local_transactions_config.contains_local_address(transaction.sender())) &&
```

Comment:
When it was  ` !self.local_transactions_config.contains_local_address(transaction.sender())) ` , received max slots error 
But this is also not correct even though tests pass